src/libraries/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -308,6 +308,7 @@ public static void RemoveReadOnlyThrowsWhenFound()
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/62713", TestPlatforms.Android)]
```

Comment:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/62713", typeof(PlatformDetection), nameof(PlatformDetection.IsAndroid), nameof(PlatformDetection.Is32BitProcess)]
```

Reply:
I'm going to merge this anyway to move forward and we can address this later.